widget/completionentry.go

```diff
@@ -136,7 +146,8 @@ type navigableList struct {
 }
 
 func newNavigableList(items []string, entry *widget.Entry, setTextFromMenu func(string), hide func(),
-	create func() fyne.CanvasObject, update func(id widget.ListItemID, object fyne.CanvasObject)) *navigableList {
+	create func() fyne.CanvasObject, update func(id widget.ListItemID, object fyne.CanvasObject),
```

Comment:
I don't understand this change - it looks like you have added another item to the parameters to put it on a new line?

Reply:
It's just my formatter automatically formatting the source code, it's a non-functional change :)

Reply:
@andydotxyz Would you prefer to undo the white space change or you're happy to merge the PR as is?

Reply:
Yes, please do put it back to not having an empty parameter which is just newline.
I will get our CI updated so that the linter runs correctly for our code standards.

Reply:
Done :)

Reply:
@andydotxyz would it be possible to get this merged soon? 🙏